README.rst

```diff
@@ -83,7 +83,15 @@ Code of Conduct
 Please note that this project is released with a `Contributor Code of Conduct`_. By
 participating in this project you agree to abide by its terms.
 
+
```

Comment:
I kind of like the strong note of ending on the Code of Conduct. What do you think is better?